### ISSUE TITLE
Can't install/update with conda 4.7.9. Not even conda itself.

### ISSUE THREAD

**Issue by shivendra90** [conda/conda#8969]
## Current Behavior

So far I've been trying to update several packages like Glue and Orange, even Spyder. But none of them would update; instead returning long error messages. This is with updating conda itself. This is the stack trace I'm getting:
```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/exceptions.py", line 1062, in __call__
        return func(*args, **kwargs)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/cli/main.py", line 84, in _main
        exit_code = do_call(args, p)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/cli/conda_argparse.py", line 82, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/cli/main_update.py", line 20, in execute
        install(args, parser, 'update')
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/cli/install.py", line 256, in install
        force_reinstall=context.force_reinstall or context.force,
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 112, in solve_for_transaction
        force_remove, force_reinstall)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 150, in solve_for_diff
        force_remove)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 262, in solve_final_state
        ssc = self._run_sat(ssc)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/core/solve.py", line 780, in _run_sat
        repodata_fn=self._repodata_fn)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/common/io.py", line 88, in decorated
        return f(*args, **kwds)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 1220, in solve
        self.find_conflicts(specs, specs_to_add, history_specs)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 336, in find_conflicts
        bad_deps = self.build_conflict_map(specs, specs_to_add, history_specs)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/resolve.py", line 418, in build_conflict_map
        bad_deps.extend(g.depth_first_search_by_name(spec, dep, sdeps[spec]))
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/models/prefix_graph.py", line 450, in depth_first_search_by_name
        matchspecs_for_chain.append(MatchSpec.merge(matchspecs_to_merge)[0])
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 467, in merge
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 467, in <lambda>
        reduce(lambda x, y: x._merge(y, union), group) if len(group) > 1 else group[0]
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 498, in _merge
        final = this_component.merge(that_component)
      File "/Users/Shiv/anaconda3/lib/python3.7/site-packages/conda/models/match_spec.py", line 751, in merge
        % (self.raw_value, other.raw_value))
    ValueError: Incompatible component merge:
      - 'eigen_py27h4f0eeca_0'
      - 'eigen_py35h4f0eeca_0'

`$ /Users/Shiv/anaconda3/bin/conda update conda`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_BACKUP_HOST=x86_64-apple-darwin13.4.0
      CONDA_BUILD_SYSROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/Users/Shiv/anaconda3/bin/conda
             CONDA_PREFIX=/Users/Shiv/anaconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/Users/Shiv/anaconda3/bin/python
               CONDA_ROOT=/Users/Shiv/anaconda3
              CONDA_SHLVL=1
     JAVA_LD_LIBRARY_PATH=/Users/Shiv/anaconda3/jre/lib/server
                     PATH=/Users/Shiv/anaconda3/bin:/Users/Shiv/anaconda3/bin:/Users/Shiv/anacon
                          da3/condabin:/Library/Frameworks/Python.framework/Versions/2.7/bin:/us
                          r/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/usr/loc
                          al/share/dotnet:/opt/X11/bin:~/.dotnet/tools:/usr/local/git/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : /Users/Shiv/anaconda3
            shell level : 1
       user config file : /Users/Shiv/.condarc
 populated config files : /Users/Shiv/.condarc
          conda version : 4.7.9
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : 
       base environment : /Users/Shiv/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/anaconda-fusion/osx-64
                          https://conda.anaconda.org/anaconda-fusion/noarch
          package cache : /Users/Shiv/anaconda3/pkgs
                          /Users/Shiv/.conda/pkgs
       envs directories : /Users/Shiv/anaconda3/envs
                          /Users/Shiv/.conda/envs
               platform : osx-64
             user-agent : conda/4.7.9 requests/2.22.0 CPython/3.7.3 Darwin/18.6.0 OSX/10.14.5
                UID:GID : 502:20
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
```

### Steps to Reproduce

```
conda update conda

# Optionally try:
conda update spyder
conda install 
```

## Expected Behavior

Packages should update/install normally.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
active environment : base
    active env location : /Users/Shiv/anaconda3
            shell level : 1
       user config file : /Users/Shiv/.condarc
 populated config files : /Users/Shiv/.condarc
          conda version : 4.7.9
    conda-build version : not installed
         python version : 3.7.3.final.0
       virtual packages : 
       base environment : /Users/Shiv/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/anaconda-fusion/osx-64
                          https://conda.anaconda.org/anaconda-fusion/noarch
          package cache : /Users/Shiv/anaconda3/pkgs
                          /Users/Shiv/.conda/pkgs
       envs directories : /Users/Shiv/anaconda3/envs
                          /Users/Shiv/.conda/envs
               platform : osx-64
             user-agent : conda/4.7.9 requests/2.22.0 CPython/3.7.3 Darwin/18.6.0 OSX/10.14.5
                UID:GID : 502:20
             netrc file : None
           offline mode : False
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /Users/Shiv/.condarc <==
ssl_verify: True
channel_priority: flexible
channels:
  - defaults
  - anaconda-fusion
report_errors: True
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /Users/Shiv/anaconda3:
#
# Name                    Version                   Build  Channel
_py-xgboost-mutex         2.0                       cpu_0    defaults
_tflow_select             2.3.0                       mkl    defaults
absl-py                   0.7.1                    py37_0    defaults
alabaster                 0.7.12                   py37_0    defaults
anaconda-client           1.7.2                    py37_0    defaults
anaconda-navigator        1.9.7                    py37_0    anaconda
anaconda-project          0.8.2                    py37_0    defaults
anyqt                     0.0.10                   py37_0    defaults
appnope                   0.1.0                    py37_0    defaults
appscript                 1.1.0            py37h1de35cc_0    defaults
asn1crypto                0.24.0                   py37_0    defaults
astor                     0.7.1                    py37_0    defaults
astroid                   2.2.5                    py37_0    defaults
astropy                   3.1.2            py37h1de35cc_0    defaults
atomicwrites              1.3.0                    py37_1    defaults
attrs                     19.1.0                   py37_1    defaults
autopep8                  1.4.4                      py_0    defaults
babel                     2.7.0                      py_0    defaults
backcall                  0.1.0                    py37_0    defaults
backports                 1.0                        py_2    defaults
backports.os              0.1.1                    py37_0    defaults
basemap                   1.2.0            py37h0acbc05_0    defaults
beakerx                   1.4.1                    py37_1    conda-forge
beautifulsoup4            4.7.1                    py37_1    defaults
blas                      1.0                         mkl    defaults
blaze                     0.11.3                   py37_0    defaults
bleach                    3.1.0                    py37_0    defaults
blis                      0.2.4                    pypi_0    pypi
blosc                     1.16.3               hd9629dc_0    defaults
bokeh                     1.1.0                    py37_0    defaults
boto                      2.49.0                   py37_0    defaults
bottleneck                1.2.1            py37h1d22016_1    defaults
bzip2                     1.0.8                h1de35cc_0    defaults
c-ares                    1.15.0               h1de35cc_1    defaults
ca-certificates           2019.5.15                     0    defaults
cctools                   895                           1    defaults
certifi                   2019.6.16                py37_0    defaults
cffi                      1.12.3           py37hb5b8e2f_0    defaults
chardet                   3.0.4                    py37_1    defaults
clang                     4.0.1                         1    defaults
clang_osx-64              4.0.1               h1ce6c1d_16    defaults
clangxx                   4.0.1                         1    defaults
clangxx_osx-64            4.0.1               h22b1bf0_16    defaults
click                     7.0                      py37_0    defaults
cloudpickle               1.2.1                      py_0    defaults
clyent                    1.2.2                    py37_1    defaults
colorama                  0.4.1                    py37_0    defaults
compiler-rt               4.0.1                hcfea43d_1    defaults
conda                     4.7.9                    py37_0    defaults
conda-package-handling    1.3.11                   py37_0    defaults
convertdate               2.1.3                   py_1000    conda-forge
cryptography              2.6.1            py37ha12b0ac_0    defaults
curl                      7.64.1               ha441bb4_0    defaults
cycler                    0.10.0                   py37_0    defaults
cymem                     2.0.2                    pypi_0    pypi
cython                    0.29.12          py37h0a44026_0    defaults
cytoolz                   0.10.0           py37h1de35cc_0    defaults
dask                      2.1.0                      py_0    defaults
dask-core                 2.1.0                      py_0    defaults
datashape                 0.5.4                    py37_1    defaults
dbus                      1.13.6               h90a0687_0    defaults
decorator                 4.4.0                    py37_1    defaults
defusedxml                0.6.0                      py_0    defaults
dill                      0.2.9                    py37_0    defaults
distributed               2.1.0                      py_0    defaults
docutils                  0.14                     py37_0    defaults
entrypoints               0.3                      py37_0    defaults
ephem                     3.7.6.0          py37h1de35cc_1    defaults
et_xmlfile                1.0.1                    py37_0    defaults
expat                     2.2.6                h0a44026_0    defaults
fast-histogram            0.7             py37h1d22016_1000    defaults
fastai                    1.0.54                   pypi_0    pypi
fastcache                 1.1.0            py37h1de35cc_0    defaults
fastprogress              0.1.21                   pypi_0    pypi
fasttsne                  0.2.13           py37h1d22016_1    defaults
fbprophet                 0.5              py37h6de7cb9_0    conda-forge
flake8                    3.7.7                    py37_0    defaults
flask                     1.0.2                    py37_1    defaults
flask-cors                3.0.8                      py_0    defaults
freetype                  2.9.1                hb4e5f40_0    defaults
gast                      0.2.2                    py37_0    defaults
geos                      3.6.2                h5470d99_2    defaults
gettext                   0.19.8.1             h15daf44_3    defaults
glib                      2.56.2               hd9629dc_0    defaults
glue-core                 0.14.2                   py37_0    defaults
glue-vispy-viewers        0.11                       py_0    defaults
glueviz                   0.13.3                        0    defaults
gmp                       6.1.2                hb37e062_1    defaults
gmpy2                     2.0.8            py37h6ef4df4_2    defaults
grpcio                    1.16.1           py37h044775b_1    defaults
h5py                      2.9.0            py37h3134771_0    defaults
hdf5                      1.10.4               hfa1e0ec_0    defaults
heapdict                  1.0.0                    py37_2    defaults
holidays                  0.9.10                     py_0    conda-forge
icu                       58.2                 h4b95b61_1    defaults
idna                      2.8                      py37_0    defaults
imageio                   2.5.0                    py37_0    defaults
imagesize                 1.1.0                    py37_0    defaults
importlib_metadata        0.17                     py37_1    defaults
intel-openmp              2019.4                      233    defaults
ipykernel                 5.1.0            py37h39e3cac_0    defaults
ipython                   7.5.0            py37h39e3cac_0    defaults
ipython_genutils          0.2.0                    py37_0    defaults
ipywidgets                7.4.2                    py37_0    defaults
isort                     4.3.21                   py37_0    defaults
itsdangerous              1.1.0                    py37_0    defaults
jdcal                     1.4.1                      py_0    defaults
jedi                      0.13.3                   py37_0    defaults
jinja2                    2.10.1                   py37_0    defaults
joblib                    0.13.2                   py37_0    defaults
jpeg                      9b                   he5867d9_2    defaults
json5                     0.8.4                      py_0    defaults
jsonschema                3.0.1                    py37_0    defaults
jupyter                   1.0.0                    py37_7    defaults
jupyter_client            5.3.1                      py_0    defaults
jupyter_console           6.0.0                    py37_0    defaults
jupyter_core              4.5.0                      py_0    defaults
jupyterlab                1.0.2                    py37_0    conda-forge
jupyterlab_server         1.0.0                      py_0    defaults
kaggle                    1.5.4                    pypi_0    pypi
keras                     2.2.4                         0    defaults
keras-applications        1.0.8                      py_0    defaults
keras-base                2.2.4                    py37_0    defaults
keras-preprocessing       1.1.0                      py_1    defaults
keyring                   18.0.0                   py37_0    defaults
keyrings.alt              3.1.1                    py37_0    defaults
kiwisolver                1.1.0            py37h0a44026_0    defaults
krb5                      1.16.1               hddcf347_7    defaults
lazy-object-proxy         1.4.0            py37h1de35cc_0    defaults
ld64                      274.2                         1    defaults
libarchive                3.3.3                h786848e_5    defaults
libcurl                   7.64.1               h051b688_0    defaults
libcxx                    4.0.1                hcfea43d_1    defaults
libcxxabi                 4.0.1                hcfea43d_1    defaults
libedit                   3.1.20181209         hb402a30_0    defaults
libffi                    3.2.1                h475c297_4    defaults
libgfortran               3.0.1                h93005f0_2    defaults
libiconv                  1.15                 hdd342a3_7    defaults
libpng                    1.6.37               ha441bb4_0    defaults
libprotobuf               3.8.0                hd9629dc_0    defaults
libsodium                 1.0.16               h3efe00b_0    defaults
libssh2                   1.8.2                ha12b0ac_0    defaults
libtiff                   4.0.10            hcb84e12_1001    defaults
libxgboost                0.80                 h0a44026_0    defaults
libxml2                   2.9.9                hf6e021a_1    defaults
llvm                      4.0.1                         1    defaults
llvm-lto-tapi             4.0.1                         1    defaults
llvm-openmp               4.0.1                hcfea43d_1    defaults
llvmlite                  0.28.0           py37h8c7ce04_0    defaults
locket                    0.2.0                    py37_1    defaults
lunardate                 0.2.0                      py_0    conda-forge
lz4-c                     1.8.1.2              h1de35cc_0    defaults
lzo                       2.10                 h362108e_2    defaults
markdown                  3.1.1                    py37_0    defaults
markupsafe                1.1.1            py37h1de35cc_0    defaults
matplotlib                3.0.3            py37h54f8f79_0    defaults
maven                     3.6.0                         0    defaults
mccabe                    0.6.1                    py37_1    defaults
mistune                   0.8.4            py37h1de35cc_0    defaults
mkl                       2019.4                      233    defaults
mkl-service               2.0.2            py37h1de35cc_0    defaults
mkl_fft                   1.0.12           py37h5e564d8_0    defaults
mkl_random                1.0.2            py37h27c97d8_0    defaults
mock                      3.0.5                    py37_0    defaults
more-itertools            7.0.0                    py37_0    defaults
mpc                       1.1.0                h6ef4df4_1    defaults
mpfr                      4.0.1                h3018a27_3    defaults
mpl-scatter-density       0.5                        py_0    defaults
mpmath                    1.1.0                    py37_0    defaults
msgpack-python            0.6.1            py37h04f5b5a_1    defaults
multipledispatch          0.6.0                    py37_0    defaults
murmurhash                1.0.2                    pypi_0    pypi
nb_conda                  2.2.1                    py37_0    defaults
nb_conda_kernels          2.2.2                    py37_0    defaults
nbconvert                 5.5.0                      py_0    defaults
nbformat                  4.4.0                    py37_0    defaults
ncurses                   6.1                  h0a44026_1    defaults
networkx                  2.3                        py_0    defaults
ninja                     1.9.0            py37h04f5b5a_0    defaults
nodejs                    11.14.0              h6de7cb9_1    conda-forge
notebook                  5.7.8                    py37_0    defaults
numba                     0.43.1           py37h6440ff4_0    defaults
numexpr                   2.6.9            py37h7413580_0    defaults
numpy                     1.16.3           py37hacdab7b_0    defaults
numpy-base                1.16.3           py37h6575580_0    defaults
numpydoc                  0.9.1                      py_0    defaults
nvidia-ml-py3             7.352.0                  pypi_0    pypi
odo                       0.5.1                    py37_0    defaults
olefile                   0.46                     py37_0    defaults
openjdk                   8.0.121          zulu8.20.0.5_0    conda-forge
openpyxl                  2.6.2                      py_0    defaults
openssl                   1.1.1c               h1de35cc_1    defaults
orange3                   3.19.0           py37h0a44026_0    defaults
packaging                 19.0                     py37_0    defaults
pandas                    0.24.2           py37h0a44026_0    defaults
pandoc                    2.2.3.2                       0    defaults
pandocfilters             1.4.2                    py37_1    defaults
parso                     0.5.0                      py_0    defaults
partd                     1.0.0                      py_0    defaults
path.py                   12.0.1                     py_0    defaults
pathlib2                  2.3.3                    py37_0    defaults
patsy                     0.5.1                    py37_0    defaults
pcre                      8.43                 h0a44026_0    defaults
pep8                      1.7.1                    py37_0    defaults
pexpect                   4.7.0                    py37_0    defaults
pickleshare               0.7.5                    py37_0    defaults
pillow                    6.1.0            py37hb68e598_0    defaults
pip                       19.1.1                   py37_0    defaults
plac                      0.9.6                    pypi_0    pypi
plotly                    3.8.1                      py_0    defaults
pluggy                    0.12.0                     py_0    defaults
pmdarima                  1.2.1                    pypi_0    pypi
preshed                   2.0.1                    pypi_0    pypi
proj4                     5.2.0                h0a44026_1    defaults
prometheus_client         0.7.1                      py_0    defaults
prompt_toolkit            2.0.9                    py37_0    defaults
protobuf                  3.8.0            py37h0a44026_0    defaults
psutil                    5.6.3            py37h1de35cc_0    defaults
ptyprocess                0.6.0                    py37_0    defaults
py                        1.8.0                    py37_0    defaults
py-xgboost                0.80             py37h0a44026_0    defaults
py4j                      0.10.8.1                 py37_0    defaults
pycodestyle               2.5.0                    py37_0    defaults
pycosat                   0.6.3            py37h1de35cc_0    defaults
pycparser                 2.19                     py37_0    defaults
pyflakes                  2.1.1                    py37_0    defaults
pygments                  2.4.2                      py_0    defaults
pylint                    2.3.1                    pypi_0    pypi
pynvx                     1.0.0                    pypi_0    pypi
pyopengl                  3.1.1a1                  py37_0    defaults
pyopenssl                 19.0.0                   py37_0    defaults
pyparsing                 2.4.0                      py_0    defaults
pyproj                    1.9.6            py37h9c430a6_0    defaults
pyqt                      5.9.2            py37h655552a_2    defaults
pyqtgraph                 0.10.0           py37h28b3542_3    defaults
pyrsistent                0.14.11          py37h1de35cc_0    defaults
pyshp                     2.1.0                      py_0    defaults
pysocks                   1.7.0                    py37_0    defaults
pystan                    2.19.0.0         py37h6440ff4_0    defaults
pytables                  3.5.2            py37h5bccee9_1    defaults
pytest                    4.4.2                    py37_0    defaults
pytest-arraydiff          0.3              py37h39e3cac_0    defaults
pytest-astropy            0.5.0                    py37_0    defaults
pytest-doctestplus        0.3.0                    py37_0    defaults
pytest-openfiles          0.3.2                    py37_0    defaults
pytest-remotedata         0.3.1                    py37_0    defaults
python                    3.7.3                h359304d_0    defaults
python-dateutil           2.8.0                    py37_0    defaults
python-libarchive-c       2.8                     py37_11    defaults
python-louvain            0.13                       py_0    defaults
python-slugify            3.0.2                    pypi_0    pypi
python.app                2                        py37_9    defaults
pytz                      2019.1                     py_0    defaults
pywavelets                1.0.3            py37h1d22016_1    defaults
pyyaml                    5.1.1            py37h1de35cc_0    defaults
pyzmq                     18.0.0           py37h0a44026_0    defaults
qt                        5.9.7                h468cd18_1    defaults
qtawesome                 0.5.7                    py37_1    defaults
qtconsole                 4.5.1                      py_0    defaults
qtpy                      1.7.1                      py_0    defaults
readline                  7.0                  h1de35cc_5    defaults
requests                  2.22.0                   py37_0    defaults
retrying                  1.3.3                    py37_2    defaults
rope                      0.14.0                     py_0    defaults
ruamel_yaml               0.15.46          py37h1de35cc_0    defaults
scikit-image              0.15.0           py37h0a44026_0    defaults
scikit-learn              0.20.3           py37h27c97d8_0    defaults
scipy                     1.2.1            py37h1410ff5_0    defaults
seaborn                   0.9.0                    py37_0    defaults
send2trash                1.5.0                    py37_0    defaults
serverfiles               0.3.0                    py37_0    defaults
setuptools                41.0.1                   py37_0    defaults
sip                       4.19.8           py37h0a44026_0    defaults
six                       1.12.0                   py37_0    defaults
snappy                    1.1.7                he62c110_3    defaults
snowballstemmer           1.9.0                      py_0    defaults
sortedcontainers          2.1.0                    py37_0    defaults
soupsieve                 1.8                      py37_0    defaults
spacy                     2.1.4                    pypi_0    pypi
sphinx                    2.0.1                      py_0    defaults
sphinxcontrib-applehelp   1.0.1                      py_0    defaults
sphinxcontrib-devhelp     1.0.1                      py_0    defaults
sphinxcontrib-htmlhelp    1.0.2                      py_0    defaults
sphinxcontrib-jsmath      1.0.1                      py_0    defaults
sphinxcontrib-qthelp      1.0.2                      py_0    defaults
sphinxcontrib-serializinghtml 1.1.3                      py_0    defaults
spyder                    3.3.4                    py37_0    defaults
spyder-kernels            0.4.4                    py37_0    defaults
sqlalchemy                1.3.5            py37h1de35cc_0    defaults
sqlite                    3.28.0               ha441bb4_0    defaults
srsly                     0.0.7                    pypi_0    pypi
statsmodels               0.10.0           py37h1d22016_0    defaults
sympy                     1.4                      py37_0    defaults
tblib                     1.4.0                      py_0    defaults
tensorboard               1.13.1           py37haf313ee_0    defaults
tensorflow                1.13.1          mkl_py37h70c3834_0    defaults
tensorflow-base           1.13.1          mkl_py37h66b1bf0_0    defaults
tensorflow-estimator      1.13.0                     py_0    defaults
termcolor                 1.1.0                    py37_1    defaults
terminado                 0.8.2                    py37_0    defaults
testpath                  0.4.2                    py37_0    defaults
text-unidecode            1.2                      pypi_0    pypi
tflearn                   0.3.2                    pypi_0    pypi
theano                    1.0.3            py37h04f5b5a_0    defaults
thinc                     7.0.5                    pypi_0    pypi
tk                        8.6.8                ha441bb4_0    defaults
toolz                     0.10.0                     py_0    defaults
torch                     1.1.0.post2              pypi_0    pypi
torchvision               0.3.0                    pypi_0    pypi
tornado                   6.0.3            py37h1de35cc_0    defaults
tqdm                      4.32.1                     py_0    defaults
traitlets                 4.3.2                    py37_0    defaults
typing                    3.7.4                    pypi_0    pypi
urllib3                   1.24.2                   py37_0    defaults
wasabi                    0.2.2                    pypi_0    pypi
wcwidth                   0.1.7                    py37_0    defaults
webencodings              0.5.1                    py37_1    defaults
werkzeug                  0.15.4                     py_0    defaults
wheel                     0.33.4                   py37_0    defaults
widgetsnbextension        3.4.2                    py37_0    defaults
wrapt                     1.11.2           py37h1de35cc_0    defaults
wurlitzer                 1.0.2                    py37_0    defaults
xlrd                      1.2.0                    py37_0    defaults
xlwings                   0.15.8                   py37_0    defaults
xz                        5.2.4                h1de35cc_4    defaults
yaml                      0.1.7                hc338f04_2    defaults
zeromq                    4.3.1                h0a44026_3    defaults
zict                      1.0.0                      py_0    defaults
zipp                      0.5.1                      py_0    defaults
zlib                      1.2.11               h1de35cc_3    defaults
zstd                      1.3.7                h5bba6e5_0    defaults
```
</p></details>

**Comment by msarahan** [comment]
This is fixed in 4.7.10, which I am uploading now.  You can download it earlier if you want from https://anaconda.org/conda-canary/conda/files

**Comment by msarahan** [comment]
PS: since conda won't update itself, you can work around by downloading this file: https://anaconda.org/conda-canary/conda/4.7.10/download/osx-64/conda-4.7.10-py37_0.tar.bz2

and then install it with:
`conda install <path to downloaded file>`

That should work, because conda bypasses a lot of the code when installing just a file.  It does not resolve dependencies, though, so don't generally rely on this way of doing things.
